include/GafferSceneUI/Private/Inspector.h

```diff
@@ -337,6 +339,9 @@ class GAFFERSCENEUI_API Inspector::Result : public IECore::RefCounted
 		/// Returns a warning associated with the plug returned
 		/// by `acquireEdit()`. This should be displayed to the user.
 		std::string editWarning() const;
+		/// Returns a user-facing description of the source of the
+		/// fallback value when `SourceType` is `Fallback`.
+		std::string fallbackDescription() const;
```

Comment:
This would be better placed just below `sourceType()`, in the queries section, since it is directly related to that function.

Reply:
Cool, makes much more sense placed there. Reordered in f59c3f9.